pkgs/applications/misc/collision/default.nix

```diff
@@ -6,6 +6,8 @@
 , desktopToDarwinBundle
 , gi-crystal
```

Comment:
You can delete gi-crystal cause we don't need this anymore

Reply:
```suggestion
```

Reply:
Oops